sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3266,16 +3253,62 @@ SIREPO.app.directive('jobsList', function(requestSender, appState, $location, $s
                 );
             }
 
+            $scope.endSimulation = function(rowIndex) {
+                let s = $scope.data.rows[rowIndex][1];
+                let r = $scope.data.rows[rowIndex][7];
+                let t = $scope.data.rows[rowIndex][0];
+                let j = {
+                    request: {simulationId: s, report: r, simulationType: t}
+                };
+                let successCallback = () => {
+                    $scope.getJobs();
+                };
+                let errorCallback = () => {
```

Comment:
I think we should add more interesting logging. It would be useful to know the request body that caused the error. Ex 
```js
const errorCallback = (request) {
  return (data) => {
    srlog`runCancel error=${data.error} from request=${request}
  }
}
...
requestSender.sendRequest('runCancel', successCallback, j.request, errorCallback(j.request));
```

Reply:
52ad443c4474fedc3a7e625ee53d4610033d3a46